### PR TITLE
Python lints

### PR DESCRIPTION
**Pull request by noirbizarre**
This PR apply the same modification than https://github.com/etalab/udata/pull/426 :
- fix the `TERRITORY_DATASETS` usage.
- fix the flake8 lints
- make use of npm run-scripts

### CODE REVIEW

File: udata_gouvfr/__init__.py [hunk 1]
```diff
@@ -9,4 +9,4 @@
 
 
 def init_app(app):
-    from . import harvesters
+    from . import harvesters  # noqa: needed fro registeration
```
Comment (davidbgk) [inline udata_gouvfr/__init__.py]: `registration`

File: udata_gouvfr/views.py [hunk 2]
```diff
@@ -8,9 +8,7 @@
 from udata.i18n import I18nBlueprint
 from udata.sitemap import sitemap
 
-from .models import (
-    DATACONNEXIONS_5_CANDIDATE, DATACONNEXIONS_6_CANDIDATE, C3, NECMERGITUR
-)
+from .models import DATACONNEXIONS_5_CANDIDATE, C3, NECMERGITUR
```
Comment (davidbgk) [inline udata_gouvfr/views.py]: How to deal with the dead code L108?

File: udata_gouvfr/harvesters/maaf.py [hunk 3]
```diff
@@ -168,15 +164,15 @@ def process(self, item):
         if metadata.get('license_id'):
             dataset.license = License.objects.get(id=metadata['license_id'])
 
-        if (metadata.get('temporal_coverage_from')
-                and metadata.get('temporal_coverage_to')):
+        if (metadata.get('temporal_coverage_from') and
```
Comment (davidbgk) [inline udata_gouvfr/harvesters/maaf.py]: Interesting, which linter prefer that indentation over the other one?

**Comment by davidbgk** [comment]
r+wc

File: udata_gouvfr/views.py (hunk 2 above)
Comment (noirbizarre) [inline udata_gouvfr/views.py]: We need to badge all candidates before using the badge as selector in the page.

File: udata_gouvfr/harvesters/maaf.py (hunk 3 above)
Comment (noirbizarre) [inline udata_gouvfr/harvesters/maaf.py]: This is W503 from http://pep8.readthedocs.org/en/latest/intro.html
(I was surprised too)

Comment (davidbgk) [inline udata_gouvfr/harvesters/maaf.py]: Alright, always cool to learn new things :)

File: udata_gouvfr/views.py (hunk 2 above)
Comment (davidbgk) [inline udata_gouvfr/views.py]: Let's create a Trello card for that.

**Pull request merged**